docs/FAQ.md

```diff
@@ -29,8 +29,14 @@ To allow that formulae to update again:
 Note that pinned, outdated formulae that another formula depends on need to be upgraded when required as we do not allow formulae to be built against non-latest versions.
 
 ## How do I uninstall old versions of a formula?
-By default, Homebrew does not uninstall old versions of a formula, so
-over time you will accumulate old versions. To remove them, simply use:
+Before `v1.9.0`, Homebrew did not uninstall old versions of a formula, so
+over time you will accumulate old versions.
+After `v1.9.0`, Homebrew introduced optional automatic cleanup via
+the `HOMEBREW_INSTALL_CLEANUP` environment variable.
+After `v2.0.0`, Homebrew's default behavior changed to automatically
```

Comment:
Let's skip all the pre-2.0.0 text; our documentation is versioned with the code accordingly.